README.md

```diff
@@ -510,6 +510,7 @@ See also [Foreign Function Interface](https://doc.rust-lang.org/book/first-editi
 * C++
   * [rust-lang/rust-bindgen](https://github.com/rust-lang/rust-bindgen) — A Rust bindings generator
   * [dtolnay/cxx](https://github.com/dtolnay/cxx) — Safe interop between Rust and C++ [<img src="https://api.travis-ci.com/dtolnay/cxx.svg?branch=master">](https://travis-ci.com/dtolnay/cxx)
+  * [rust-cpp](https://crates.io/crates/cpp) - Embed C++ code directly in Rust.
```

Comment:
There's a CI build badge on the actual code repository for this. Can you please add that to the end of the line here?

Reply:
Done.